types/yup/index.d.ts

```diff
@@ -193,6 +232,13 @@ export interface ValidateOptions {
     context?: object;
 }
 
+export interface TestMessageParams {
```

Comment:
Another addition, this basically lists out the properties that I found in the code.